templates/jaasai/index.html

```diff
@@ -51,7 +51,15 @@ <h1>Big software, tamed</h1>
         </div>
       </div>
       <div class="col-6 u-vertically-center">
-        <img src="https://assets.ubuntu.com/v1/f3f75945-home-promo-bundle.svg" alt="Pictogram of bundle" style="margin-bottom: -2rem;" />
+        {{
+          image(
+            url="https://assets.ubuntu.com/v1/f3f75945-home-promo-bundle.svg",
+            alt="Pictogram of bundle",
+            width="600",
+            height="570",
+            hi_def: True,
```

Comment:
This needs to be `hi_def=True`. Where did you copy this from? I couldn't see it in [the README](https://github.com/canonical-web-and-design/canonicalwebteam.image-template/blob/master/README.rst).